OpenRA.Game/Server/Server.cs

```diff
@@ -154,16 +154,14 @@ public Server(IPEndPoint endpoint, ServerSettings settings, ModData modData)
 				}
 			};
 
-			FieldLoader.Load(LobbyInfo.GlobalSettings, modData.Manifest.LobbyDefaults);
-
-			foreach (var t in serverTraits.WithInterface<INotifyServerStart>())
-				t.ServerStarted(this);
-
-			Log.Write("server", "Initial mod: {0}", ModData.Manifest.Mod.Id);
-			Log.Write("server", "Initial map: {0}", LobbyInfo.GlobalSettings.Map);
-
 			new Thread(_ =>
 			{
+				foreach (var t in serverTraits.WithInterface<INotifyServerStart>())
+					t.ServerStarted(this);
```

Comment:
This ends up running `LobbyCommands.LoadMap()`. Is it safe do to that off the main thread?